vagrant.nodes.yaml

```diff
@@ -0,0 +1,37 @@
+---
+defaults:
+  box: ubuntu/trusty64
+  memory: 3072
+
+nodes:
```

Comment:
These are AWS node names (eg `db-admin`), not Carrenza names. Are we going the AWS way with the vagrant setup? 🚀

Reply:
That was my thinking, yes - make the Vagrant dev environment representative of the future...